js/models/messages.js

```diff
@@ -1693,12 +1693,36 @@
       });
 
       if (!found) {
+        // Exponential backoff, giving up after 5 attempts:
+        if (attemptCount < 5) {
+          setTimeout(() => {
+            window.log.info(
+              `Looking for the message id : ${id}, attempt: ${attemptCount + 1}`
+            );
+            this.copyFromQuotedMessage(message, attemptCount + 1);
+          }, attemptCount * attemptCount * 500);
+        }
+
         quote.referencedMessageNotFound = true;
```

Comment:
I guess so, with this PR we show the error and remove it as soon as we find the message (hopefully after 500-1000ms). I think it would be wrong to show that everything is OK at first and then (after 5 attempts or so) "oh wait, actually it wasn't". Also, if we close the app before the last attempt (so we didn't set the error), it won't ever get the chance to show this error.